src/pages/commits/ViewToggle.tsx

```diff
@@ -0,0 +1,97 @@
+import { useQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import {
+  SegmentedControl,
+  SegmentedControlOption,
+} from "@leafygreen-ui/segmented-control";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Body } from "@leafygreen-ui/typography";
+import Icon from "components/Icon";
+import { size } from "constants/tokens";
+import {
+  ProjectHealthView,
+  ProjectHealthViewQuery,
+  ProjectHealthViewQueryVariables,
+} from "gql/generated/types";
+import { PROJECT_HEALTH_VIEW } from "gql/queries";
+import { useQueryParam } from "hooks/useQueryParam";
+import { ProjectFilterOptions } from "types/commits";
+
+type Props = {
+  hasFilters: boolean;
+  identifier: string;
+  viewFilter: ProjectHealthView;
```

Comment:
The prop logic for this is a little confusing. It accepts a prop which is value from a query param. But it also reads the same query param on #L31. I think this prop could be omitted.